PyInstaller/compat.py

```diff
@@ -870,10 +870,13 @@ def check_requirements():
         if 'win32api' in sys.modules or 'pywintypes' in sys.modules:
             # Users should never see this error; if it occurs, it means someone
```

Comment:
What about running check_requirements as soon as compat is imported? That would solve the problem and ensure that developers wouldn't need to worry about this in the future?

Reply:
The problem was circular imports of varying annoyance, so it seemed easier to not deal with them at the time. However, I've changed my mind and now I've changed it to force pywintypes et al to be retrieved from the compat module, which mostly solves the circular import issue.